HW02/Alexey_Zalyotov/hw_02_t_03/lib/duration_calculator.rb

```diff
@@ -0,0 +1,15 @@
+require 'time'
+
+class DurationCalculator
+  def initialize(timestamps)
+    @timestamps = timestamps
+  end
+
+  def total
+    durations = []
+    @timestamps.each_cons(2) do |pair|
+      durations << (pair[1].to_time.to_f - pair[0].to_time.to_f).round(1)
+    end
```

Comment:
I'm not sure but maybe we can use `.map` here?
```suggestion
    @timestamps.each_cons(2).map do |pair|
      (pair[1].to_time.to_f - pair[0].to_time.to_f).round(1)
    end
```

Reply:
I did it. But it didn`t work. Now it`s ok. Magic...